guidelines/releases-guidelines/02-kyma-release-process.md

```diff
@@ -70,64 +67,28 @@ Follow these steps to release another Kyma version.
 
 3. Push the branch to the `test-infra` repository.
 
-4. Create a PR to `test-infra/release-x.y`. This triggers the pre-release job for `watch-pods`. 
-
-   > **NOTE:** To trigger the `watch-pods` build without introducing any changes, edit any file within the `test-infra` repository and create a pull request. You don't need to merge it as the job is triggered anyway. After a successful `watch-pods` image build, close the pull request.
-
-5. Update the `RELEASE_VERSION` file with the name of the next minor release candidate and merge the pull request to `master`. For example, if the `RELEASE_VERSION` on the `master` branch is set to `1.4.2`, then change the version to `1.5.0-rc1`.
+4. Update the `RELEASE_VERSION` file with the name of the next minor release candidate and merge the pull request to `master`. For example, if the `RELEASE_VERSION` on the `master` branch is set to `1.4.2`, then change the version to `1.5.0-rc1`.
 
 ### kyma-project/kyma
 
 1. Inside the release branch do the following changes.
 
-   i. Update your PR with the version and the directory of components used in `values.yaml` files.
+   i. In `installation/resources/installer.yaml` replace `eu.gcr.io/kyma-project/develop/installer:{image_tag}` with `eu.gcr.io/kyma-project/kyma-installer:{release_version}`
 
-   Find these values in the files:
+   ii. In `tools/kyma-installer/kyma.Dockerfile` replace:
 
-   ```yaml
-   dir: develop/
-   version: {current_version}
    ```
-  
-   ```yaml
-   dir: pr/
-   version: {current_version}
+   ARG INSTALLER_VERSION="{kyma_operator_version}"	
+   ARG INSTALLER_DIR={kyma_operator_path}
+   FROM $INSTALLER_DIR/kyma-operator:$INSTALLER_VERSION
    ```
 
-   Replace them with:
+   with
 
-   ```yaml
-   dir:
-   version: {release_version}
    ```
```

Comment:
I'm not sure dividing the sentence with the codeblocks is the best way to structure this. I'd stick to the way it was formatted before changes, i.e.:
```
  ii. In `tools/kyma-installer/kyma.Dockerfile` find these values:
```
...
```
Replace them with: 
```
...